### PR TITLE
Enable importing SSL certificates in the Docker container.

### PR DESCRIPTION
**Pull request by gijskant**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -1,10 +1,10 @@
 FROM openjdk:8-jre-alpine
 
-ENV APP_PORT 8083
-ENV GB_BACKEND_VERSION 0.4.0-RC1
+ARG GB_BACKEND_VERSION="1.0.0"
```
Comment (ewelinajozwik) [inline docker/Dockerfile:3]: why `ARG`? It is also used in `entrypoint.sh`, it probably won't be resolved there correctly.

Comment (gijskant) [reply docker/Dockerfile:3]: The main reason is that ARG can be overridden from the command line with `docker build --build-arg GB_BACKEND_VERSION=1.0.1 .`, whereas ENV variables cannot be changed from the command line.
But I still need to test these changes, so I'll check if the `entrypoint.sh` script still works.

Comment (gijskant) [reply docker/Dockerfile:3]: I now fixed it.

**Review by ewelinajozwik — APPROVED** [review]
nice, many useful improvements!

**Pull request merged**